src/js/components/Diagram/stories/typescript/Connections.tsx

```diff
@@ -1,13 +1,13 @@
 import React from 'react';
 import { storiesOf } from '@storybook/react';
-import isChromatic from 'chromatic/isChromatic';
 
 import { Gremlin, IceCream } from 'grommet-icons';
 import { Stack, grommet, Grommet, Box, Diagram } from 'grommet';
 
 import { DiagramConnectionAnchor, DiagramConnectionType } from '../../index';
 
 // Type annotations can only be used in TyoeScript files
+// Remove ':DiagramConnection...'
```

Comment:
Can you add the full message that addresses the non-TS users

Reply:
Yes, I think you were looking at an older version of this PR when you added this comment. This was done in the "updated comment" commit on Monday